server/routes/reports.test.ts

```diff
@@ -57,4 +57,37 @@ describe('GET /reports/external-movements', () => {
         expect(res.text).toContain('31/01/2023')
       })
   })
+
+  it('should render correct paging URL when filtered', () => {
+    return request(app)
+      .get('/reports/external-movements?filters.direction=in')
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        expect(res.text).toContain(
+          '?selectedPage=2&amp;pageSize=20&amp;sortColumn=prisonNumber&amp;sortedAsc=true&amp;filters.direction=in',
```

Comment:
Why is the selected page here 2?

Reply:
This is the link to the second page, from the front page, as discussed on Slack (repeated here in case anyone else reads this later).